frontend/src/core/codemirror/cm.ts

```diff
@@ -87,6 +88,8 @@ export const setupCodeMirror = (opts: CodeMirrorSetupOpts): Extension[] => {
     // Cell editing
     cellMovementBundle(cellId, cellMovementCallbacks, hotkeys),
     cellCodeEditingBundle(cellId, cellCodeCallbacks, hotkeys),
+    // Markdown autorun
+    markdownAutoRunExtension(cellMovementCallbacks),
```

Comment:
Would this be better in the MarkdownLanguageAdapter class? Better separation of concerns and no need to do to the if check